### ISSUE TITLE
Start from curlconverter when implementing curl => Dart

### ISSUE THREAD

**Issue by csells** [kiruto/curl#1]
I added [Dart support to the curlconverter project](https://github.com/NickCarneiro/curlconverter/blob/master/generators/dart.js) that generates http client code from various languages. You can start there if you're implementing the same functionality in Dart.

**Comment by techouse** [comment]
@csells Good idea :) The URL's changed somwhat since that JS file has been renamed to a TS file 🤪 https://github.com/curlconverter/curlconverter/blob/master/src/generators/dart.ts
